modules/jaggeryapps/ml/site/project/compare.jag

```diff
@@ -323,7 +383,126 @@
             var analysisName = modelName.split(".")[0];
             window.location.href = '../analysis/view-model.jag?analysisId='+analysisId+'&datasetId='+datasetId+'&modelId='+modelId+'&projectName='+projectName+'&analysisName='+analysisName+'&modelName='+modelName;
         });
-    }        
+    }
+
+    /* pagination plugin */
+    $.fn.pageMe = function(opts){
+        var $this = this,
```

Comment:
Isn't this pageMe function is a common function that other pages may use? If so can we move it to the common JS file? @cdathuraliya I remember you introduced a similar method?